Documentation/ApiOverview/BackendLayout/Index.rst

```diff
@@ -0,0 +1,211 @@
+.. include:: ../../Includes.txt
+
+.. _be-layout:
+
+==============
+Backend Layout
+==============
+
+Since TYPO3 4.5 there has been a database record type the "Backend Layout" to define a combination of rows and columns
+to which content can be added in the page module.
+
+With TYPO3 7.4 a new feature was introduced to define backend layouts in TYPO3 via PageTSConfig. It implements a
+generic PageTS provider for backend layouts to make backend layouts reusable across installations.
+
+.. _be-layout-video:
+
+Backend Layout Video
+================================
+
+Benji: How to implement frontend layouts in TYPO3 using backend layouts
+
+.. youtube:: RoHaeo4fq34
+
+
+.. _be-layout-definition:
+
+Backend Layout Definition
+=========================
+
+Backend Layouts can be configured either as "Backend Layout" record in a sysfolder or as PageTsConfig entry in
+ :typoscript:`mod.web_layout.BackendLayouts`. Each layout will be saved with a key. The Backend Layout records are
+using their uid as a key, therefore layouts defined via PageTsConfig should use a non-numeral String key. It is a good
+practise to use a descriptive name as key.
+
+The entries title and icon are being used to display the Backend Layout options in the page properties.
+
+The overall grid size will be defined by :typoscript:`config.backend_layout.colCount` and :typoscript:`rowCount`.
+Additional rows in the :typoscript:`rows` Array and additional columns in the each rows :typoscript:`columns` section
+will be ignored when they are greater then :typoscript:`rowCount` or :typoscript:`colCount` respectively.
+
+Each column position can span several columns and or several rows. Each column position must have a distinct number
+between 0 and n assigned. It is best practise to always assign "0" to the main column if there is such a thing as a
+main column. Multiple Backend Layouts that contain similar parts, i.e. header, footer, aside, ... should within one
+project have the same number assigned each. This simplifies sliding up to find content within one column position to
+use content of a parent page across layouts.
```

Comment:
Instead of "... should within one project have the same number assigned each." I would write ".. should each have assigned the same number within one project". 

"This simplifies sliding up to find content within one column position to
use content of a parent page across layouts." -> I think this is too specific. I would keep it general and write something like: "This leads to a uniform position of the content, which makes it more clear for further use."